content/logs/aws.md

```diff
@@ -0,0 +1,133 @@
+{{< img src="logs/aws/adding_trigger.png" alt="Adding trigger" >}}
+
+Select the log source and then follow the AWS instructions: 
+{{< img src="logs/aws/integration_lambda.png" alt="Integration Lambda" >}}
+
+For instance for S3 Buckets, do not forget to set the correct event type:
+{{< img src="logs/aws/object_created.png" alt="Object Created" >}}
+
+#### ELB
+
+Elastic Load Balancing provides access logs that capture detailed information about requests or connections sent to your load balancer. Each log contains information such as the time it was received, the client's IP address, latencies, request paths, and server responses. Use these access logs to analyze traffic patterns and troubleshoot issues.
+Elastic Load Balancing publishes a log file for each load balancer node every 5 minutes. The load balancer can deliver multiple logs for the same period.
+Add ELB logs to Datadog to:
+
+* Check if your services and apps are well-balanced
+* Know in real-time, what is the min, max, average and 95th percentile response time for each services and apps running
+* Be notified when specific IPs have a anormal behavior (malicious attempts, too many failure, etc...)
+* Determine how many connection peaks you have? and when did they occurred?
+* See how bots are going through your webpages (for SEO purposes for instance)
+
+
+ELB logs are written in a s3 bucket and consumed by a lambda function.
+Enable the logging on your ELB first to collect your logs:
+{{< img src="logs/aws/configure_access_logs.png" alt="Configure Access Logs" >}}
+
+Set interval to **5 minutes** and define your s3 buckets:
+{{< img src="logs/aws/s3_location.png" alt="S3 Location" >}}
+
+Then go back to the Lambda function and define a trigger on the corresponding s3 bucket.
+
+#### CloudTrail
+
+AWS CloudTrail is an audit service. Use AWS CloudTrail to get a history of AWS API calls and related events for your account. This includes calls made by using the AWS Management Console, AWS SDKs, command line tools, and higher-level AWS services.
+
+You can identify which users and accounts called AWS for services that support CloudTrail, the source IP address from the calls, and when the calls occurred.
+You can integrate CloudTrail into applications using the API, automate trail creation for your Organization, check the status of your trails, and control how administrators turn CloudTrail logging on and off.
+
+When you define your Trails, you can select a s3 bucket to write the log in:
+{{< img src="logs/aws/tail_s3_selection.png" alt="S3 Selection" >}}
+
+You can send link the lambda function to this s3 bucket to send your logs.
+
+
+#### ECS
+
+ECS logs are the legacy Docker container. They are not directly related to the ECS service, but they correspond to the logs written by the running app (in the Docker Container). 
+
+Collect ECS logs directly from the containers thanks to our [Docker integration](/integrations/docker_daemon) (recommended).
```

Comment:
thanks to our `Agent 6` Docker integration. 

And let's remove the recommended parenthesis.

Reply:
👍